packages/manager/modules/manager-pci-common/src/components/quantity-selector/QuantitySelector.component.tsx

```diff
@@ -49,116 +35,67 @@ export function QuantitySelector({
   ...props
 }: Readonly<QuantitySelectorProps>) {
   const { t } = useTranslation('pci-quantity-selector');
-  const [, setCounter] = useState(0);
   const error = {
     min: min !== undefined && value < min,
     max: max !== undefined && value > max,
     nan: Number.isNaN(value),
   };
-  const isValid = !error.min && !error.max && !error.nan;
-  const forceRedraw = () => setCounter((i) => i + 1);
-
-  // forced redraw to fix ODS quantity not refreshing
-  useEffect(forceRedraw, [min, max]);
 
   return (
-    <OsdsFormField className={className} inline {...props}>
+    <OdsFormField className={className} {...props}>
       <div className={contentClassName}>
         <div slot="label" className="flex gap-2">
           {label && (
-            <OsdsText
-              color={ODS_THEME_COLOR_INTENT.text}
-              size={ODS_THEME_TYPOGRAPHY_SIZE._200}
-            >
+            <OdsText preset="span" className="text-[14px]">
               {label}
-            </OsdsText>
+            </OdsText>
           )}
           {labelHelpText && (
-            <OsdsPopover className="w-4 h-4">
-              <OsdsIcon
-                slot="popover-trigger"
-                name={ODS_ICON_NAME.HELP}
-                size={ODS_ICON_SIZE.xxs}
-                className="cursor-help"
-                color={ODS_THEME_COLOR_INTENT.text}
-              />
-              <OsdsPopoverContent>{labelHelpText}</OsdsPopoverContent>
-            </OsdsPopover>
+            <>
+              <div id="popover-trigger">
+                <OdsIcon
+                  name={ODS_ICON_NAME.question}
+                  className="cursor-help text-[16px]"
+                />
+              </div>
+              <OdsPopover triggerId="popover-trigger" className="w-4 h-4">
```

Comment:
does having multiple quantity selector with the same triggerId in the same page would work ?

Reply:
@frenautvh 
Yep, that's a good catch, I juste pasted from the documentation but It's better to rename it 👍🏻